### ISSUE TITLE
Auto-indent changes multiline strings

### ISSUE THREAD

**Issue by trbrc** [rollup/rollup#164]
The auto-indent feature adds indentation to the contents of multiline strings.
```javascript
export const myTemplateString = `
Template string content
`;

export const myEscapedString = "\
Escaped string content\
";
```

`$ rollup -f iife -n strings strings.js`

```javascript
(function (exports) { 'use strict';

	const myTemplateString = `
	Template string content
	`;

	const myEscapedString = "\
	Escaped string content\
	";

	exports.myTemplateString = myTemplateString;
	exports.myEscapedString = myEscapedString;

})((this.strings = {}));
```

A workaround would be to use the `indent: false` option, but as far as I can tell that's not possible with the CLI. Since the defaults are `indent: true`, rollup is currently subtly breaking such code out of the box. A better solution would be to make the auto-indent feature treat these cases as an exception.

I suggest that the output should be:

```javascript
(function (exports) { 'use strict';

	const myTemplateString = `
Template string content
`;

	const myEscapedString = "\
Escaped string content\
";

	exports.myTemplateString = myTemplateString;
	exports.myEscapedString = myEscapedString;

})((this.strings = {}));
```

**Comment by Victorystick** [comment]
That's definitely a bug. Thanks for reporting it. I'm not sure how to best fix the general case of indented template strings, but we should make sure to have a CLI option, such as `--no-indent`.

**Comment by Rich-Harris** [comment]
[magic-string does allow you](https://github.com/Rich-Harris/magic-string#sindent-prefix-options-) to pass an `exclude` option to `magicString.indent(...)`, for exactly this reason. I just had a quick look now and it looks like there'll be a bit of an art to rigging it up, but it's definitely solvable. We should expose that CLI option anyway though

**Comment by Rich-Harris** [comment]
Fixed by 4fcbba63e040b4afe33d84d0ff7fd6b50c0e9daf, which I've rolled into #163 – will release as 0.18.0 as soon as I've tied down one loose end

**Comment by timshannon** [comment]
Works great.

Thanks,
